src/zope/traversing/namespace.py

```diff
@@ -672,8 +688,10 @@ def traverse(self, name, ignored):
                 # TODO: I am not sure this is the best solution.  What
                 # if we want to enable tracebacks when also trying to
                 # debug a different skin?
```

Comment:
BTW I think this old TODO is no longer relevant.  I'm too lazy to dig through git history, but I think ++debug++errors originally did a directlyProvides(request, skin) before it was changed to directlyProvides(request, povidedBy(request) + skin), at which point the old TODO should have been resolved.

Reply:
Thank you. 

The comment and the `directlyProvides() + providedBy()` is present in the first version of the file found in this repo. It's also present in the earliest version still present on PyPI, 3.4-dev_r72940-py2.4.

Reply:
s/too lazy to dig through git history/too lazy to dig through svn history/ :)